nncf/onnx/graph/nncf_graph_builder.py

```diff
@@ -153,26 +344,28 @@ def create_nncf_graph(onnx_model: onnx.ModelProto) -> NNCFGraph:
         nncf_graph = NNCFGraph()
         onnx_graph = ONNXGraph(onnx_model)
         for node in onnx_graph.get_all_nodes():
-            metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node.op_type)
-            if metatype.get_subtypes():
-                subtype = metatype.determine_subtype(onnx_model, node)
-                if subtype is not None:
-                    metatype = subtype
-
-            if metatype in WEIGHT_LAYER_METATYPES:
-                is_shared = onnx_graph.is_node_shared(node)
-                weight_edge_name = onnx_graph.get_weight_tensor_edge(node)
-                edge = onnx_graph.get_edge(weight_edge_name)
-                weight_shape = ONNXGraph.get_edge_shape(edge)
-                layer_attributes = ONNXExtendedLayerAttributes(node.input, node.output, weight_shape)
-            else:
-                is_shared, weight_edge_name, layer_attributes = None, None, None
+            metatype = get_metatype(onnx_graph, node)
+            port_ids = _get_weight_port_ids(node, onnx_graph)
+            is_shared = None
+            weight_attrs = {}
+            node_attrs = _get_node_attrs(node, onnx_graph)
+            bias_attrs = _get_bias_attr(node, onnx_graph)
+            if port_ids:  # If node has weight
```

Comment:
Let's call it `weight_port_ids`, because `port_ids` also exists for nodes with activations, I would say.

Reply:
Done